tests/unit_tests/testing/test_helpers.py

```diff
@@ -67,3 +67,73 @@ def test_new_testeff_version_file_not_found(builtins_open):
                 FileNotFoundError, match="The testeff executable could not be found."
             ):
                 helpers._new_testeff_version()
+
+
+def test_skip_multiple_version_test_single_version():
+    config = {"CONFIGURATION": {"MODEL_VERSION": "v1"}}
```

Comment:
I know it doesn't matter...just note that we are using semantic versioning everywhere...

Reply:
You are right, this is something I have complained about myself as well. I changed it.